code/go/internal/validator/folder_item_spec.go

```diff
@@ -93,3 +94,9 @@ func (s *folderItemSpec) validate(fs http.FileSystem, folderSpecPath string, ite
 	}
 	return errs
 }
+
+func refreshFormatCheckersContext(currentPath string) {
```

Comment:
Given that this is a global instruction, should we just move it into `init` to make that more explicit/obvious? It will also guarantee that the `relative-path` format checker is initialized exactly once at package initialization time.

Reply:
Let me describe the case here:

The method `refreshFormatCheckersContext` is called just before calling `Validate` to modify the global state every time it's executed against a package, because there is no way to sneak a context (with a package path).

Reply:
Ah yes, of course! I totally missed the `currentPath` argument 🤦.

Okay, I understand what's going on here now and why you have to do it this way. This is good as-is but I made a related comment at the location where this function is being called from.